woocommerce/payment-gateway/class-sv-wc-payment-gateway-payment-form.php

```diff
@@ -977,7 +977,10 @@ public function render_payment_fields() {
 	 */
 	protected function render_payment_field( $field ) {
 
-		woocommerce_form_field( $field['name'], $field, $field['value'] );
+		if ( isset( $field['name'] ) ) {
```

Comment:
@adrianostanley I think this prevents the fields from https://github.com/skyverge/wc-plugins/pull/4121/files from showing up in the form. Wouldn't it?

I realize I didn't specify that on the document, but the intention is to remove the name attribute to prevent the browser from sending the value in those fields to the server. We still need those fields to get the values with JavaScript and pass them to Checkout.js

Reply:
@wvega oh, got it now, sorry.

Done in https://github.com/skyverge/wc-plugin-framework/pull/515/commits/9946f43b43dbf509714e6b0e7d45ce14050807e9.

It won't affect the other PR that was already merged, so we're good here.

Thank you for reviewing.